src/controller.cpp

```diff
@@ -602,12 +605,16 @@ bool Controller::processNextStep() {
   }
 
   if (nextStep == BackendFailure) {
+    emit vpn->recordGleanEvent(GleanSample::backendFailureError);
```

Comment:
2 issues here:
-  this is not the only place where we emit `readyToBackendFailure`.
- When the `readyToBackendFailure` signal is emitted, MozillaVPN changes state to `StateBackendFailure`. And we already have events for this state. I wonder if we need this separate event.

Reply:
Okay, that makes sense I'll remove this event.